trk/src/Screens/ClimbCreate/index.js

```diff
@@ -80,51 +80,81 @@ const ClimbInputData = (props) => {
         height: 400,
         cropping: true,
       });
-      setImage(pickedImage.path);
+
+      // Save the full image path for later uploading
+      const imagePath = pickedImage.path;
+
+      // Set the image state to include the full path
+      setImage({ path: imagePath });
+
     } catch (err) {
       console.error("Error picking image:", err);
     }
   };
 
 
+  const uploadImage = async (imagePath) => {
+    const filename = imagePath.split('/').pop().replace(/\.jpg/gi, "").replace(/-/g, "");
+    const storageRef = storage().ref(`climb_images/${filename}`);
+    await storageRef.putFile(imagePath);
+    return { id: filename, path: storageRef.fullPath, timestamp: new Date().toISOString() };
+  };
+
+
```

Comment:
I had to find a way to associate a unique id to each new image upload (or edit). Thankfully, ImagePicker automatically generates one for us when it is used. I make some adjustments to that full path string to only keep the unique ID. 

For example: keeping A86C4245-6F0A-4B5D-B4C0-F6DD2AB0D9E8, instead of "/private/var/mobile/Containers/Data/Application/0024CF5E-550C-4316-91D0-D1596956496F/tmp/react-native-image-crop-picker/A86C4245-6F0A-4B5D-B4C0-F6DD2AB0D9E8.jpg"

Also removing the "-" and the ".jpg"